consensus/mock/consensusRatingStub.go

```diff
@@ -0,0 +1,28 @@
+package mock
+
+// ConsensusRatingStub -
+type ConsensusRatingStub struct {
```

Comment:
change the name rating to something else - this is a local view of the node

Reply:
It would be PeerHonestyStub